geode-core/src/main/java/org/apache/geode/internal/cache/RegionStats.java

```diff
@@ -135,7 +135,9 @@
 
   void incEvictWorkTime(long delta);
 
-  void incClearCount();
+  void incBucketClearCount();
+
+  void incRegionClearCount();
```

Comment:
This method should have remained as `incClearCount()`, I think, to remain consistent with `CachePerfStats`.

Reply:
RegionPerfStats and other classes which used "incClearCount()" have been replaced with "incRegionClearCount()" as that is more descriptive and you're right - they should be consistent. This was being blocked previously by the error you identified in your previous comment.